File: exo1/main.java
class Main {
    class public static void main(String[] args) {
    Book newBook1 = new Book("Le petit poucet", 9.99)
    System.out.print(newBook1.bookName, newBook1.bookPrice);
    }
}

class Book {
   String bookName;
   int bookPrice;

   Book(String bookName, int bookPrice){
       this.bookName = bookName;
       this.bookPrice = bookPrice;
   }
}

File: exo1/main.ts
class Book {
    bookName: string;
    bookPrice: number;

    constructor(bookName: string, bookPrice: number) {
        this.bookName = bookName,
            this.bookPrice = bookPrice;
    }
}

let book01 = new Book("le petit poucet", 9.99)

console.log(book01)
